Tasks/AzureWebAppContainerV1/README.md

```diff
@@ -34,13 +34,13 @@ The task is used to deploy a Web  project to an existing Azure Web App or Functi
 
 * **Azure Subscription\*:** Select the AzureRM Subscription. If none exists, then click on the **Manage** link, to navigate to the Services tab in the Administrators panel. In the tab click on **New Service Endpoint** and select **Azure Resource Manager** from the dropdown.
 
-* **App Service Name\*:** Select the name of an existing Azure App Service. Enter the name of the Web App if it was provisioned dynamically using the [Azure PowerShell task](https://github.com/Microsoft/vsts-tasks/tree/master/Tasks/AzurePowerShell) and [AzureRM PowerShell scripts](https://msdn.microsoft.com/en-us/library/mt619237.aspx).
+* **App Service Name\*:** Select the name of an existing Azure App Service. Enter the name of the Web App if it was provisioned dynamically using the [Azure PowerShell task](https://github.com/microsoft/azure-pipelines-tasks/tree/master/Tasks/AzurePowerShellV5) and [AzureRM PowerShell scripts](https://docs.microsoft.com/en-us/powershell/module/az.websites/?view=azps-5.0.0#app_service).
```

Comment:
In the links do not include /en-us/ as it might affect the localization. Plz correct the doc URLS

Reply:
done